documentation/CMakeLists.txt

```diff
@@ -12,25 +12,27 @@ CPMAddPackage(
 )
 
 CPMAddPackage(
-  NAME StableCoder-cmake-scripts
-  GITHUB_REPOSITORY StableCoder/cmake-scripts
-  GIT_TAG 3d2d5a9fb26f0ce24e3e4eaeeff686ec2ecfb3fb
+  NAME MCSS
+  GITHUB_REPOSITORY mosra/m.css
+  GIT_TAG e6eff549fb5edeabacf01369d6b845a2a59c2ebe
+  DOWNLOAD_ONLY YES
 )
 
 # ---- Doxygen ----
 
-set(BUILD_DOCUMENTATION ON CACHE INTERNAL "")
-include(${StableCoder-cmake-scripts_SOURCE_DIR}/doxygen.cmake)
+find_package(Doxygen)
 
 # set Doxyfile variables
 set(DOXYGEN_PROJECT_NAME Greeter)
 set(DOXYGEN_PROJECT_VERSION ${Greeter_VERSION})
 set(DOXYGEN_PROJECT_ROOT "${CMAKE_CURRENT_LIST_DIR}/..")
-
-# see https://github.com/StableCoder/cmake-scripts#doxygen-doxygencmake for additional options
-build_docs(
-  TARGET_NAME GenerateDocs
-  DOXYFILE_PATH ${CMAKE_CURRENT_LIST_DIR}/Doxyfile
-  OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR}/doxygen
-  PROCESS_DOXYFILE ON
-)
+set(DOXYGEN_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/doxygen")
+
+if (DOXYGEN_FOUND)
```

Comment:
imo we don't need the `find_package(Doxygen)` and the `if (DOXYGEN_FOUND)` guard.
- We are not using any of the targets or variables defined in the FindDoxygen module
- When building on a system without Doxygen, the error message given by mcss should be more clear

Reply:
Done.